src/schemas/lwm2m-types-versions.spec.ts

```diff
@@ -0,0 +1,36 @@
+import {
+	ConnectivityMonitoring_4_urn,
+	Device_3_urn,
+	Humidity_3304_urn,
+	Location_6_urn,
+	parseURN,
+	Pressure_3323_urn,
+	Temperature_3303_urn,
+} from './index.js'
```

Comment:
I can do it but just to point out that all the URN  in the project must change to be imported from `/schemas/index.js` and not from `@nordicsemiconductor/lwm2m-types`

Reply:
Yes, this is the intention here. To clearly document that we are re-using definitions from another library.

Reply:
Imports were updated https://github.com/MLopezJ/asset-tracker-lwm2m-js/pull/7/commits/3bc425bc297bb23d0bafc9f9576d562c9751bc9e